package.json

```diff
@@ -18,7 +18,7 @@
     "examples:integration:python": "test/run-against-dist tools/build-examples.sh python",
     "examples:integration:typescript": "test/run-against-dist tools/build-examples-sequential.sh typescript",
     "examples:integration:go": "test/run-against-dist tools/build-examples-sequential.sh go",
-    "pretest": "prettier --check .",
+    "lint": "prettier --check .",
```

Comment:
We should also execute eslint here, shouldn't we?

Reply:
Yeah, good point - Probably gonna try this Action https://github.com/marketplace/actions/lint-action